### PR TITLE
Add link to AM Backende API endpoint and docs

### PR DESCRIPTION
**Pull request by jamesscottbrown**

### CODE REVIEW

File: ui/routes.py [hunk 1]
```diff
@@ -305,6 +305,15 @@ def on_get(self, _: falcon.Request, resp: falcon.Response, worker_doc: str):
             raise falcon.HTTPNotFound(title="Docs not found", description="'{}' is not available".format(worker_doc))
 
 
+class BackendDetails:
```
Comment (oserban) [inline ui/routes.py]: The name of the class does not adhere to the general naming convention. Please rename this into BackendDetailsView

File: ui/__init__.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ def setup_ui(logging_level: str = "debug", backend_url: str = "http://localhost:
     app.add_route('/api/store/{store_name}/project/{project_name}/asset/{asset_name}/upload', UploadApi(controller=_controller))
     app.add_route('/api/store/{store_name}/project/{project_name}/asset/{asset_name}/process/{worker_type}', WorkerApi(controller=_controller))
     app.add_route('/api/store/{store_name}/project/{project_name}/asset/{asset_name}/files', FilesApi(controller=_controller))
+    app.add_route('/backend', BackendDetails(controller=_controller))
```
Comment (oserban) [inline ui/__init__.py]: This is not an api route so it should be placed along with the other views

File: ui/routes.py (hunk 1 above)
Comment (jamesscottbrown) [reply ui/routes.py]: Ok - I have also modified ``WorkerDocs`` to match this convention

File: ui/__init__.py [hunk 3]
```diff
@@ -37,6 +37,7 @@ def setup_ui(logging_level: str = "debug", backend_url: str = "http://localhost:
     app.add_route('/view/store/{store_name}/project/{project_name}/edit', ProjectEdit(controller=_controller))
     app.add_route('/view/store/{store_name}/project/{project_name}/asset/{asset_name}', AssetEdit(controller=_controller))
     app.add_route('/view/store/{store_name}/project/{project_name}/object/{object_name}', ObjectEdit(controller=_controller))
+    app.add_route('/backend', BackendDetailsView(controller=_controller))
```
Comment (oserban) [inline ui/__init__.py]: the route has to be renamed as well into view/backend

**Pull request merged**